components/ws-manager-bridge/src/bridge.ts

```diff
@@ -332,6 +332,12 @@ export class WorkspaceManagerBridge implements Disposable {
             instance.status.podName = instance.status.podName || status.runtime?.podName;
             instance.status.nodeIp = instance.status.nodeIp || status.runtime?.nodeIp;
             instance.status.ownerToken = status.auth!.ownerToken;
+            instance.status.metrics = {
+                image: {
+                    totalSize: status.metadata.metrics?.image?.totalSize,
```

Comment:
:bubbles: Would it make sense try to maintain existing DB values if there already are some?

Reply:
Added in [`f4f7a27` (#20426)](https://github.com/gitpod-io/gitpod/pull/20426/commits/f4f7a27a656e8a2e41dfcefb729aca8c5f5e9ed9)